\documentclass[10pt, openright,oneside]{book}
\usepackage[a4paper,width=150mm,top=25mm,bottom=25mm]{geometry}
\usepackage{csquotes} %fixes some error

\usepackage[english]{babel}
\usepackage{subfiles}
\usepackage{float} % keepd tables etc in place
\usepackage{wrapfig}

% Removes paragraph indentation (not needed most of the time now)
\usepackage{parskip}

% Provides ways to make nice looking tables
\usepackage{booktabs}
% Typesets URLs sensibly - with tt font, clickable in PDFs, and not breaking across lines
\usepackage{url}
\usepackage{hyperref}
\hypersetup{
    colorlinks=true,
    linkcolor=black,
    citecolor=gray,
    filecolor=magenta,      
    urlcolor=gray,
}
 
\usepackage{mwe}% for putting 2 images side by side
\usepackage[font={footnotesize},labelfont=bf]{caption}

\usepackage[usenames, dvipsnames]{color} %text color

\usepackage[table,xcdraw]{xcolor}
\usepackage{multirow}

%helps add a blank page
\usepackage{afterpage}
\newcommand\blankpage{%
    \null
    \thispagestyle{empty}%
    \addtocounter{page}{-1}%
    \newpage}
    

%abbreviations management 
\usepackage{nomencl}
\makenomenclature
\renewcommand*{\nompreamble}{\small}
\renewcommand{\nomname}{List of Abbreviations}

% for adding pretty headings
%\usepackage{fancyhdr}
%\pagestyle{fancy}

\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\graphicspath{ {images/} }
\usepackage{wallpaper}

\usepackage[backend=biber,style=ieee, doi=false,isbn=false]{biblatex}
\addbibresource{mendeley.bib}
\addbibresource{nonmenrefs.bib}

%allows subsub sections and paragraphs
\setcounter{tocdepth}{4}
\setcounter{secnumdepth}{4}

%\usepackage{refcount} %for making lists of fig appear in contents
\usepackage[nottoc]{tocbibind}
\begin{document}


\frontmatter
%%  Begin title page
%%  ==================================================================
    \thispagestyle{empty}
    \ULCornerWallPaper{1}{ku-coverpage/nat-farve.pdf}
    \ULCornerWallPaper{1}{ku-coverpage/nat-en.pdf}
    %\begin{adjustwidth}{-1cm}{-0.5cm}
    
    \vspace*{4.5cm} 
    \noindent \textbf{\huge Identification of autophagy signatures\\ in breast cancer using\\ The Cancer Genome Atlas data} \\
    \\[0.5cm]
    \textit{\Large Master's Thesis} \\
    [0.5cm]
    \textbf{\Large MSc Bioinformatics} \\
    \vspace*{.1cm} \\
%     {\huge Subtitle} \\
    \begin{tabbing}
    % adjust the hspace below for the longest author name
    \textbf{Marina Vabistsevits} \hspace{1cm} \= \texttt{<rzq995@alumni.ku.dk>} \\
    \\[9cm]
    \textbf{\Large Supervisors} \\
    Dr Elena Papaleo \> \texttt{<elenap@cancer.dk>} \\
    Prof Albin Sandelin \> \texttt{<albin@binf.ku.dk>} \\
    Dr Kristoffer Vitting-Seerup\> \texttt{<kristoffer.vittingseerup@bio.ku.dk>}
    \\[1cm]
    
    \end{tabbing}
    \begin{center}
    \textit{7 August, 2017} 
    \end{center}
    %\end{adjustwidth}

    \newpage
    \ClearWallPaper
    
%%  ==================================================================
%%  End title page



\pagestyle{plain}




\afterpage{\blankpage} %adds an empty page

\chapter{Abstract}
\input{chapters/abstract}


\tableofcontents

\cleardoublepage
\addcontentsline{toc}{part}{\small List of Abbreviations}
\addtocontents{toc}{\protect\vspace{-0.1em}}
%abbreveations
\printnomenclature[6em]
\hfill % add blank space

\nomenclature{AJCC}{The American Joint Committee on Cancer }
\nomenclature{CBL}{Computational Biology Lab}
\nomenclature{CPM}{Counts per million}
\nomenclature{DCRC}{Danish Cancer Research Centre}
\nomenclature{DE}{Differential expression / differentially expressed }
\nomenclature{DEA}{Differential expression analysis }
\nomenclature{DEGs}{Differentially expressed genes }
\nomenclature{EDA}{Exploratory data analysis}
\nomenclature{ER}{Estrogen receptor}
\nomenclature{GDC}{Genomic Data Commons }
\nomenclature{HER2}{Human epidermal growth factor receptor 2 }
\nomenclature{ICD-O-3}{International Classification of Diseases for Oncology, 3rd edition}
\nomenclature{IDC}{Invasive ductal carcinoma }
\nomenclature{IHC}{Immunohistochemistry}
\nomenclature{ILC}{Invasive lobular carcinoma}
\nomenclature{NCI}{National Cancer Institute }
\nomenclature{NHGRI}{National Human Genome Research Institute}
\nomenclature{PCA}{Principal component analysis }
\nomenclature{PCs}{Principal components}
\nomenclature{PR}{Progesterone receptor}
\nomenclature{RPKM}{Reads per kilobase of transcript per million}
\nomenclature{FPKM}{Fragments per kilobase of transcript per million}
\nomenclature{TCGA}{The Cancer Genome Atlas}
\nomenclature{TCGA-BRCA}{The Cancer Genome Atlas breast cancer data}
\nomenclature{TNM}{Tumour, Nodes, Metastasis (system)}
\nomenclature{UICC}{International Union Against Cancer }
\nomenclature{WHO}{World Health Organisation }
\nomenclature{TNBC}{Triple Negative Breast Cancer }
\nomenclature{qRT-PCR}{Quantitative reverse transcriptase polymerase chain reaction }
\nomenclature{PAM50}{Prediction Analysis of Microarray, based on 50 gene set}
\nomenclature{DCC}{Data Coordinating Centre}
\nomenclature{PAS}{Phagophore assembly site}
\nomenclature{PE}{Phosphatidylethanolamine}
\nomenclature{ATG}{Autophagy-related gene}
\nomenclature{PI3P}{Phosphatidylinositol 3-phosphate}
\nomenclature{LIR}{LC3-interacting region}
\nomenclature{ROS}{Reactive oxygen species}
\nomenclature{PPI}{Protein-protein interactions}
\nomenclature{STRING}{Search Tool for the Retrieval of Interacting Genes/Proteins (database)}

\listoffigures
\listoftables

\mainmatter


\chapter{Introduction}
\input{chapters/introduction}

\chapter{Methods}
\input{chapters/methods}

\chapter{Results}
\input{chapters/results}

\chapter{Discussion}
\input{chapters/discussion}



\cleardoublepage
\addcontentsline{toc}{part}{\small Acknowledgements}
\addtocontents{toc}{\protect\vspace{-1.5em}}
\chapter*{Acknowledgements}

I would like to thank the Computational Biology Laboratory at the Danish Cancer Research Centre for allowing me to join their group to write my Masterâ€™s thesis with them as a part of my degree at the University of Copenhagen.  

I am very thankful to Dr Elena Papaleo, the group leader, for her professional guidance and valuable support throughout the project. 

I would also like to express my gratitude to Dr Kristoffer Vitting-Seerup, who provided internal supervision of my project. His ability to explain difficult bioinformatics concepts and show how to correctly use the tools to answer biological questions has been greatly appreciated and has enabled me to complete high quality work.

I would also like to thank Prof Albin Sandelin for agreeing to supervise this project, and more importantly, for making the MSc courses on high-throughput data analysis that precede the thesis part of the degree extremely interesting,  as this is what motivated me to do my thesis in this research field. 

Finally, I would like to acknowledge and thank the colleagues at DCRC (in the Cell Death & Metabolism and Cell Stress & Survival Units)  for providing a list of expert-curated autophagy genes to use in this project.





\cleardoublepage
\addcontentsline{toc}{part}{\small Bibliography}
\addtocontents{toc}{\protect\vspace{-0.2em}}
\printbibliography



\cleardoublepage
\addcontentsline{toc}{part}{\small Appendices}
\addtocontents{toc}{\protect\vspace{-0.2em}}

\chapter*{Appendices}


   \textbf{Appendix A. }One dimensional PCA plots for year and sample source data, based on cancer samples only.\\
   
   
   \textbf{Appendix B. }Raw counts of differentially expressed genes in the models with a single main effect (PAM50, stages, morphology).\\
   
   
   \textbf{Appendix C. }Enrichment analysis results on differentially expressed genes data.\\
   
   
   \textbf{Appendix D. }Enrichment analysis results on soft-clustering data for individual PAM50 subtypes with the cluster figures.\\
   
   
   \textbf{Appendix E. } Venn Diagrams of Luminal B, HER2-enriched, Normal-like subtypes showing the overlap between downregulated autophagy genes in different cancer stages. \\







\backmatter
\end{document}
